libs/feature/catalog/src/lib/organisations/organisations.service.ts

```diff
@@ -50,12 +49,30 @@ export class OrganisationsService {
     private groupService: GroupService
   ) {}
 
-  normalizeName(name: string): string {
-    return name
-      .normalize('NFD') // decompose graphemes to remove accents from letters
-      .replace(/[\u0300-\u036f]/g, '') // remove accent characters
-      .toLowerCase()
-      .replace(/[^a-z0-9]/g, '') // replace all except letters & numbers
+  compareNormalizedStrings(
```

Comment:
compare function often return -1, 0, 1 in order to sort things.
it's more an equal test here